src/main/electron-context-menu.ts

```diff
@@ -35,12 +35,7 @@ const removeUnusedMenuItems = (
     | electron.MenuItem
 
   return menuTemplate
-    .filter(
-      menuItem =>
-        menuItem !== undefined &&
-        menuItem !== false &&
-        menuItem.visible !== false
-    )
+    .filter(menuItem => menuItem !== undefined && menuItem.visible !== false)
```

Comment:
This currently breaks after this change. Apparently `menuItem` can sometimes be a `boolean`, even though this is not given in the TypeScript definition

Reply:
this is a hack to make this syntax/style of menu item composing work:
```js
[
condition && MenuItem,
condition && MenuItem,
MenuItem,
]
```